src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -41,6 +44,7 @@ public class AddCommand extends Command {
     public static final String MESSAGE_DUPLICATE_FIELD = "This person contains duplicate fields"
             + "(Email, Phone, Telegram, or Github) with another person.";
 
+    private static final Logger logger = Logger.getLogger("Foo");
```

Comment:
Shouldn't this be Logger.getLogger(AddCommand.class)

Reply:
Fixed, thanks!